firefox-ios/firefox-ios-tests/Tests/StorageTests/TestBrowserDB.swift

```diff
@@ -54,7 +54,15 @@ class TestBrowserDB: XCTestCase {
         let results = db.runQuery("SELECT bmkUri, title FROM bookmarksLocal WHERE type = 1",
                                   args: nil,
                                   factory: { row in
-            (row[0] as! String, row[1] as! String)
+            guard let bmkUri = row[0] as? String else {
+                return ("", "")
```

Comment:
Let's use in this case `XCTFail` too in order to be clear that something went wrong with casting.